### DISCUSSION TITLE
Universal Windows App

### DISCUSSION THREAD

**Discussion by JMvanderDuim** [CoreWCF/CoreWCF#925] [Q&A]
I have an universal windows app and need to add WCF service connectivity. But there is no CoreWCF template in VisualStudio Enterprise to do so. If I add the templates to the project an error occurs. 
Can this be done if yes, how?

**Comment by Kralizek** [comment]
_not a maintainer_

I think you should look at the [client library](https://github.com/dotnet/wcf). CoreWCF is focusing on the server bits.
